src/Sdk/DTPipelines/Pipelines/ObjectTemplating/PipelineTemplateConverter.cs

```diff
@@ -264,5 +263,351 @@ internal static class PipelineTemplateConverter
 
             return result;
         }
+
```

Comment:
I assume these changes are copied from the existing server code?

Reply:
Yes